app/views/layouts/map.html.erb

```diff
@@ -46,12 +46,17 @@
 
     <% unless @user %>
       <div class="welcome">
-        <h2><a><span class="icon close"></span></a><%= t 'layouts.intro_header' %></h2>
+        <h2><%= t 'layouts.intro_header' %></h2>
+        <div class="close-wrap"><span class="icon close"></span></div>
```

Comment:
What's the reason for the extra close-wrap wrapping here and all the other related changes? I'm particularly worried about some of the changes that move things from matching on close to matching on close-wrap but I'm not sure if they are currently matching other things as well...